### ISSUE TITLE
Template Operator: Developer Makefile Tooling: spin up a minimalistic Control Plane Development Environment

### ISSUE THREAD

**Issue by jakobmoellersap** [kyma-project/kyma-operator#107]

**Comment by jakobmoellersap** [comment]
Additional Discussions resulted in these points:

- For dedicated E2E Tests, we are not planning to replace them. Instead it should be a quick way to test if your operator is compliant with the Kyma Control Plane Environment.
- It could be sufficient to just check on the Contract that each Module must implement (in this case we only need the `status.state` object as is described in the `template-operator`. That way we wouldn't need a full control plane

**Comment by khlifi411** [comment]
> * It was noticed that while `kyma` CLI is currently responsible for k3d-style single-cluster deployments, multi-cluster deployments (with a control-plane) would then not be managed by kyma-cli. We should think about at least providing a `kyma provision`command that would solve the scaffolding of the environment, similarly to how `kyma provision k3d` is bootstrapping a runtime environment now.

+1 for `kyma alpha provision control-plane` and `kyma alpha provision runtime` CLI commands to provision two `k3d` clusters for `kcp` and `skr` respectively to be used for developer tests.

**Comment by tobiscr** [comment]
@jakobmoellersap : is this ticket still required, assume it could be covered https://github.com/kyma-project/cli/issues/1318

**Comment by jakobmoellersap** [comment]
agree, this is related to #226 and we should target to instead call kyma cli only

**Comment by pbochynski** [comment]
Q3 is over. Please update the timeline.

**Comment by jakobmoellersap** [comment]
This has all been completed in CLI commands and the new template-operator. A control-plane can be spun up via guide. We dont need a separate guide
